Savings-API/Controllers/SavingsController.cs

```diff
@@ -28,28 +30,28 @@ public IActionResult GetSaving(int id)
             return Ok(saving);
         }
 
-        [HttpGet("get-all")]
-        public IActionResult GetAllSavings()
+        [HttpGet()]
```

Comment:
Tutaj możesz zamienić `[HttpGet()]` na [HttpGet]`. Gdy w atrybutach nie podajesz parametrów, to można pominąć nawias.